### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by K-Shao**

### CODE REVIEW

File: portfolio/src/main/webapp/programming.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  </head>
+  <body>
+    <div>
+      <script src="nav.js"></script>
+    </div>
+    <p>In terms of programming, I love building everything, but I especially love games. 
+       One of my first big projects was a game of Chess, and some of my later projects have 
+       followed in a similar vein. I'm also really curious about Machine Learning, despite
+       not having any formal instruction in it, and I'm always ready to learn! For some of
+       my prouder projects, check out my <a href="http://github.com/K-Shao">Github</a>! </p>
```
Comment (christophervo-g) [inline portfolio/src/main/webapp/programming.html:17]: nit: this is pretty long, so it might be better to separate out the paragraph tags into separate lines

File: portfolio/src/main/webapp/nav.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+document.write('\
+  <div class="topnav">\
+    <a class="active" href="index.html">Home</a>\
+    <a href="hobbies.html">Hobbies</a>\
+    <a href="programming.html">Programming</a>\
+  </div> \
+  '
+);
```
Comment (christophervo-g) [inline portfolio/src/main/webapp/nav.js:9]: Is this file necessary? Would it be possible to insert this directly into the index.html page?

Comment (K-Shao) [reply portfolio/src/main/webapp/nav.js:9]: The idea here was to be able to cleanly insert the nav bar into each individual page so I can edit the nav bar once for changes everywhere. Linh suggested the idea, but I'm not sure if I implemented it like she intended.

Comment (christophervo-g) [reply portfolio/src/main/webapp/nav.js:9]: Got it, this is good then. My only suggestion then would be to separate the raw string into a variable to reference inside of document.write()

File: portfolio/src/main/webapp/navbar.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<div class="topnav">
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/navbar.html:1]: Are you currently using this file somewhere? Or is this a placeholder you're adding before you make other changes?

File: portfolio/src/main/webapp/nav.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const content = '\
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/nav.js:1]: Nice job with the nav bar. This uses purely javascript, but if you'd like to use jQuery it's possible to do this as well: https://medium.com/@AmyScript/how-to-reduce-reuse-and-recycle-your-code-389e6742e4ac

My only reservation with this is that now you have some html that is in a js file instead of an html file. I'm not too sure if this is considered a style issue, might be worth asking a virtual support engineer if there's a way for you to do so if you have time! If you do, would love to hear what they tell you about this.

File: portfolio/src/main/webapp/navbar.html (hunk 3 above)
Comment (K-Shao) [reply portfolio/src/main/webapp/navbar.html:1]: I'm hoping to write some CSS next, so I wanted some way to be able to reference it.

File: portfolio/src/main/webapp/nav.js (hunk 4 above)
Comment (K-Shao) [reply portfolio/src/main/webapp/nav.js:1]: Will do!
